### PR TITLE
Mark late applications as late

### PR DESCRIPTION
**Pull request by kotlarz**
This closes https://github.com/webkom/committee-admissions/issues/23

image
image

### CODE REVIEW

**Comment by kotlarz** [comment]
@JonasBak CSV should be fixed now.

File: frontend/src/routes/ApplicationForm/FormStructure.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ const FormStructure = ({
           {myApplication && (
             <StyledSpan bold>
               <Moment format="dddd Do MMMM , \k\l. HH:mm:ss">
-                {myApplication.time_sent}
+                {myApplication.created_at}
```
Comment (JonasBak) [inline frontend/src/routes/ApplicationForm/FormStructure.js]: Shouldn't this be `updated_at`? As it's the last time the application was registered

Comment (JonasBak) [inline frontend/src/routes/ApplicationForm/FormStructure.js]: This value doesn't exist the first time you submit, without reloading the page
image

Comment (LudvigHz) [reply frontend/src/routes/ApplicationForm/FormStructure.js]: Yup, fixed

Comment (LudvigHz) [reply frontend/src/routes/ApplicationForm/FormStructure.js]: Fixed with rebase.

**Pull request merged**